src/components/interfaces/HMI_API.xml

```diff
@@ -3657,6 +3678,9 @@
     <param name="dynamicUpdateCapabilities" type="DynamicUpdateCapabilities" mandatory="false">
       <description>Contains the head unit's capabilities for dynamic updating features declaring if the module will send dynamic update RPCs.</description>
     </param>
+    <param name="keyboardCapabilities" type="KeyboardCapabilities" mandatory="false">
```

Comment:
```suggestion
    <param name="keyboardCapabilities" type="Common.KeyboardCapabilities" mandatory="false">
```

But it seems like WindowCapability and KeyboardCapabilities both are included in the Common interface, so probably proposed changes are redundant